ui/src/main/webapp/src/app/executions/executions-list.component.html

```diff
@@ -39,14 +39,19 @@ <h2 i18n="heading|executions list">Analysis</h2>
         </td>
         <td>{{execution.timeStarted | date: 'short'}}</td>
         <td>
-            <a class="link cancel" *ngIf="canCancel(execution)" (click)="cancelExecution(execution)" i18n="button">Cancel</a>
+            <a class="pointer link cancel" *ngIf="canCancel(execution)" (click)="cancelExecution(execution)" i18n="button">Cancel</a>
             <!-- left for when dynamic report will be linked again
             <a
                     class="link"
                     *ngIf="execution.state == 'COMPLETED'"
                     [routerLink]="['/projects', execution.projectId, 'reports', execution.id]">View Reports</a>
             -->
-            <a *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports" class="link external-link" target="_blank" href="{{formatStaticReportUrl(execution)}}">Static Report</a>
+            <a *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports" class="pointer link" target="_blank" href="{{formatStaticReportUrl(execution)}}">
```

Comment:
Well, with it being an icon, I couldn't really use that, and the mockup requests and icon.